app/models/git_hub_user.rb

```diff
@@ -15,8 +15,12 @@ def login
   # Public
   #
   def admin_organization_memberships
+    no_cache_headers = { 'Cache-Control' => 'no-cache, no-store' }
```

Comment:
TIL the user agent can send the `Cache-Control` header. It looks like the middleware *is* [handling this correctly](https://github.com/plataformatec/faraday-http-cache/blob/master/lib/faraday/http_cache/request.rb#L23). I stand corrected.